### ISSUE TITLE
Craft URL Field Error when no valid URL

### ISSUE THREAD

**Issue by davidhellmann** [angell-co/Spoon#53]
When you have a normal Craft CMS URL Field and you type in a not valid url and save the entry the following error will be shown in the console and the UI is broken. 

```
FieldManipulator.min.js?v=1550150921:1 Uncaught TypeError: Cannot read property 'addClass' of undefined
    at s.constructor.<anonymous> (FieldManipulator.min.js?v=1550150921:1)
    at i (jquery.js:496)
```

https://media.discordapp.net/attachments/456442884258922529/552793783599890453/unknown.png?width=1369&height=407

https://media.discordapp.net/attachments/456442884258922529/552793850419347456/unknown.png?width=1369&height=658

It's on different installations the same problem.

**Comment by joshangell** [comment]
Hmm, OK I really can’t reproduce this - I just checked. Please could you send me your composer.json and a DB backup to josh@angell.io and I’ll get it sorted for you asap :)

**Comment by joshangell** [comment]
Also, try clearing caches and force reloading, see if that solves it.

**Comment by davidhellmann** [comment]
Caches cleared hard reload but problem is existing. DB is a bit to big. 

But here is the composer.json
```json
{
  "name": "craftcms/craft",
  "description": "Craft CMS",
  "keywords": [
    "craft",
    "cms",
    "craftcms",
    "project"
  ],
  "license": "MIT",
  "homepage": "https://craftcms.com/",
  "type": "project",
  "support": {
    "email": "support@craftcms.com",
    "issues": "https://github.com/craftcms/cms/issues",
    "forum": "https://craftcms.stackexchange.com/",
    "source": "https://github.com/craftcms/cms",
    "docs": "https://craftcms.com/docs",
    "rss": "https://craftcms.com/changelog.rss"
  },
  "minimum-stability": "dev",
  "prefer-stable": true,
  "require": {
    "aelvan/craft-cp-element-count": "^1.0",
    "aelvan/imager": "^2.1",
    "angellco/spoon": "^3.2",
    "craftcms/cms": "3.1.12",
    "craftcms/element-api": "^2.5",
    "craftcms/mailgun": "^1.3",
    "craftcms/redactor": "2.3.1",
    "doublesecretagency/craft-cpcss": "^2.1",
    "ether/logs": "^3.0",
    "fruitstudios/linkit": "1.1.8",
    "haariga/craft-gonzo": "1.0.0-beta.1",
    "luwes/craft3-codemirror": "^1.0",
    "marionnewlevant/twig-perversion": "2.0.7",
    "mildlygeeky/craft-kint": "1.0.0",
    "mmikkel/child-me": "^1.0",
    "mmikkel/cp-clearcache": "^1.0",
    "mmikkel/cp-field-inspect": "^1.0",
    "monachilada/craft-matrixtoolbar": "^1.0",
    "nystudio107/craft-cookies": "^1.1",
    "nystudio107/craft-imageoptimize": "^1.5.3",
    "nystudio107/craft-minify": "^1.2",
    "nystudio107/craft-seomatic": "3.1.42",
    "nystudio107/craft-typogrify": "^1.1",
    "pennebaker/craft-architect": "^2.2",
    "solspace/craft3-freeform": "2.5.11",
    "studioespresso/craft-dumper": "^1.3",
    "topshelfcraft/environment-label": "^3.1",
    "topshelfcraft/supersort": "^3.0",
    "verbb/expanded-singles": "^1.0",
    "verbb/image-resizer": "^2.0",
    "verbb/super-table": "2.1.7",
    "vlucas/phpdotenv": "^2.4.0"
  },
  "autoload": {
    "psr-4": {
      "modules\\": "modules/"
    }
  },
  "config": {
    "sort-packages": true,
    "optimize-autoloader": true,
    "platform": {
      "php": "7.0"
    }
  },
  "scripts": {
    "post-root-package-install": [
      "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
    ]
  }
}
```

**Comment by joshangell** [comment]
OK, I really can’t reproduce this without a DB. Please do send one.

**Comment by joshangell** [comment]
Fixed in 94082aef3a1ed3812f904693b40100665a2c5bc0
